net-libs/libquotient/libquotient-0.7.2-r1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Qt-based SDK to develop applications for Matrix"
+HOMEPAGE="https://github.com/quotient-im/libQuotient"
+SRC_URI="https://github.com/quotient-im/libQuotient/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/libQuotient-${PV}"
+
+LICENSE="LGPL-2+"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
```

Comment:
You'll have to either drop all but ~amd64 in this new revision for now, and then we can file a rekeywording bug (because of the new deps not being keyworded), or we make the new deps optional. 

What makes more sense here?

(Does it make sense to make this optional for libquotient? I'm not a user so I don't know.)

Reply:
I don't think it makes sense to make it optional.

The biggest selling point of Matrix is that it's a federated E2EE protocol, so it seems useless to not have E2EE included, especially when the encryption is done through the official `olm` library.

As such, I've removed the keywords in the new one and will open a rekeywording bug later. But I will also accept whatever decision as long as I can have it on `amd64`.